bitnami/spark/templates/hpa-worker.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.worker.enableAutoscaling }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ include "spark.name" . }}
+    helm.sh/chart: {{ include "spark.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/component: worker-autoscaler
+  name: {{ include "spark.fullname" . }}-autoscaler
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "spark.fullname" . }}-worker
+  minReplicas: {{ .Values.worker.replicaCount }}
+  maxReplicas: {{ .Values.worker.replicasMax }}
+  metrics:
+  - type: Resource
+    resource:
```

Comment:
Would it make sense to have this section included as a templated section inside the yaml file? Such as affinity, nodeSelector, etc. I do not know if we want to use other metrics rather than cpu

Reply:
Yes, it could have more sense.

Reply:
Done

Reply:
Could you change it to template?

Reply:
Could you change it to template?

Reply:
Could you use tpl instead?

Reply:
Ping